packages/vscode-extension/src/handlers.ts

```diff
@@ -892,7 +907,11 @@ export async function checkPermission(env: string): Promise<boolean> {
     result =
       (teamsAppPermission.roles?.includes("Administrator") ?? false) &&
       (aadPermission.roles?.includes("Owner") ?? false);
+    ExtTelemetry.sendTelemetryEvent(Stage.checkPermission, {
```

Comment:
same as above. Also another question, why telemetry is not sent in grant permission?

Reply:
By the way, seems log is not added?

Reply:
processResult function at end will send the telemetry and show error dialog, so not send here